course_catalog/task_helpers_test.py

```diff
@@ -41,8 +41,24 @@ def get_micromasters_data(mocker):
     """
     Mock function to retrieve micromasters data
     """
-    with open("./test_json/test_micromasters.json", "r") as test_data:
-        micromasters_data = json.load(test_data)
+    micromasters_data = [
```

Comment:
If you put this function in `course_catalog.conftest.py` it will be available to all tests in the app and you won't need another copy in `tasks_test.py`

Reply:
Done.